classes/class06/twotter/public/javascripts/main.js

```diff
@@ -5,24 +5,88 @@ var onError = function(data, status) {
 	console.log("error", data);
 };
 
-
-
-//LOGGING IN
-var $loginform = $("#login");
-var onSuccessOut = function(data, status) {
-	$('#new_username')
-	.not(':button, :submit, :reset, :hidden')
-	.val('')
-	.removeAttr('checked')
-	.removeAttr('selected');
-};
-$loginform.submit(function login(event){
+//MAKING A TWOTE
+var $newForm = $("#new_tool");
+$newForm.submit(function newTwote(event){
 	event.preventDefault();
-	var postData = {username: $('#new_username').val()};
-	$.post("ingredients", postData)
-		.done(onSuccessOut)
+	var postData = {text:$("#twote_text").val(), 
+					name:$(".loggedInUser").attr("name")};
+	console.log(postData);
+
+	$.post("newTwote", postData)
+		.done(onSuccessNewTwote)
 		.error(onError);
 });
+var onSuccessNewTwote = function(data, status){
+	//make new twote appear
+	console.log("making twote");
+	var twote_disp = Handlebars.templates['twote_disp'];
+	console.log(twote_disp(data));
+	$('.twotesl').prepend(twote_disp(data));
+	//$('#twote_user').prepend(twote_disp(data.user));
+
+}; 
+
+var $users = $('.users');
+// var id = $('.users').attr('id').toString();
+// var $userHighlight = $("#"+id);
+// console.log($userHighlight);
+$users.click(function highlight(event){
+	event.preventDefault();
+	// console.log($(this).attr("twotes"));
+	// var twotes = $(this).attr('id');
+	//var $twotesUsers = $('.twote_user');
+
+
+	var name = undefined;
+	for(i=0;i< $users.length; i++){
+		name = $(this).attr('id');
+	}
+	console.log(name);
+	console.log($("." + name));
+	$("." + name).parent().toggleClass('highlight');
+	$(this).toggleClass('highlight');
+
+	//$("#twote_user").toggleClass('highlight');
+
+	// console.log(id);
+	// var id = $("#twote_user").attr("id")
+	// $("." + id).toggleClass('highlight');
+
+	// for(i=0; i< $twotesUsers.length; i++){
+	// 	console.log($($twotesUsers[i]).attr('id'));
+	// }
+
+	// var postData = {clickedUserId: $(this).attr('id')};
+	
+	//$.post("highlight", postDat)
+
+});
+
+//LOGGING IN
+// var $loginform = $("#login");
+// var onSuccessOut = function(data, status) {
+
+// 	console.log($('div'));
+
+// 	$('#welcome').html(data.name);
+// 	$('#username')
+// 	.not(':button, :submit, :reset, :hidden')
+// 	.val('')
+// 	.removeAttr('checked')
+// 	.removeAttr('selected');
+
+
+// };
+// $loginform.submit(function login(event){
+// 	event.preventDefault();
+// 	console.log($('#username').val());
+// 	var postData = {username: $('#username').val()};
+	
+// 	$.post("login", postData)
+// 		.done(onSuccessOut)
+// 		.error(onError);
+// });
```

Comment:
The great thing about Git is that when you use it there's no need to comment out code in a commit -- you can just delete it! It'll be in earlier commits if you ever need it.